doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -1208,6 +1208,10 @@ div.install > input:checked + label {
   display: none;
 }
 
+pre.sk-expandable > span:before {
```

Comment:
wouldn't it be possible to use `pre.sk-expandable:before` directly and get rid of the `span` tags?

Or would it put the prompt symbol out of the `pre` box?

Reply:
You need to identify what counts as a new line. 
similarly, you can have a look at generated HTML outpu, `span` elements are used for this purpose.

Reply:
Indeed, I misread the code and thought there was one `pre` per line. Sorry for the noise.